AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -300,12 +300,15 @@ - (void)testLogsNotFlushedImmediatelyWhenIntervalIsCustom {
   // Then
   [self waitForExpectationsWithTimeout:kMSTestTimeout
                                handler:^(NSError *error) {
-                                 OCMVerify([self.sut startTimer:OCMOCK_ANY]);
```

Comment:
I've seen this in several places. Definitely this isn't correct and we should make sure to verify a mock instance instead of a real instance.